src/Elastic.Apm/Api/IExecutionSegment.cs

```diff
@@ -102,6 +102,15 @@ public interface IExecutionSegment
 			Dictionary<string, Label> labels = null
 		);
 
+		/// <summary>
+		/// Captures a log line as an APM error.
```

Comment:
Should we adopt `log event` to describe a log?

Reply:
Yes. Seems MS docs also use `log event`. Changed it.